scripts/e2e.npm.publish.sh

```diff
@@ -21,11 +21,11 @@ fi
 # To model publication correctly, this script needs to run
 # without web3's dev deps being installed. It installs
 # what it needs here.
-npm install -g verdaccio@4.4.4
+npm install -g verdaccio@5.18.0
 npm install -g npm-auth-to-token@1.0.0
-npm install -g lerna@^3.20.2
-npm install -g typescript@^3.9.5
-npm install -g webpack@^4.44.1 webpack-cli@^3.3.12 clean-webpack-plugin@^3.0.0
```

Comment:
Could you please tell about why `clean-webpack-plugin` has been removed? Does it conflicts with Lerna 6? Or is this unrelated cleaning?
Thanks,

Reply:
Hi there, in the last [PR](https://github.com/web3/web3.js/pull/5649/files#diff-1fb26bc12ac780c7ad7325730ed09fc4c2c3d757c276c3dacc44bfe20faf166fL3) I merged webpack-plugin was removed because we now use the clean flag in webpack.config